sdks/java/container/build.gradle

```diff
@@ -86,51 +76,21 @@ licenseReport {
   renderers = [new JsonReportRenderer()]
 }
 
-def imageJavaVersion = project.hasProperty('imageJavaVersion') ? project.findProperty('imageJavaVersion') : '8'
-docker {
-  name containerImageName(
-          name: "${project.docker_image_default_repo_prefix}java${imageJavaVersion}_sdk",
-          root: project.rootProject.hasProperty(["docker-repository-root"]) ?
-                  project.rootProject["docker-repository-root"] :
-                  project.docker_image_default_repo_root,
-          tag: project.rootProject.hasProperty(["docker-tag"]) ?
-                  project.rootProject["docker-tag"] : project.sdk_version)
-  dockerfile project.file("./Dockerfile")
-  files "./build/"
-  buildArgs([
-    'pull_licenses': project.rootProject.hasProperty(["docker-pull-licenses"]) ||
-                     project.rootProject.hasProperty(["isRelease"]),
-    'java_version': imageJavaVersion,
-  ])
+task buildAll {
+  dependsOn ':sdks:java:container:java8:docker'
+  dependsOn ':sdks:java:container:java11:docker'
 }
 
 task pullLicenses(type: Exec) {
+  dependsOn generateLicenseReport
+  generateLicenseReport.outputs.cacheIf { false }
```

Comment:
(not a blocker for this PR) @kennknowles can you comment on why we need this? It would be good to have a comment explaining it. Looks like it was added in https://github.com/apache/beam/commit/ebd65e5314eb42131eace3be62390b3570e71292